keyboards/keycapsss/plaid_pad/config.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+/* VID & PID from vusb project, see tmk_core/protocol/vusb/USB-IDs-for-free.txt"*/
+#define VENDOR_ID 0xFEED
+#define PRODUCT_ID 0x7983
```

Comment:
```suggestion
#define PRODUCT_ID 0xAF12
```

```
echo -n "Plaid-Pad" | shasum
```

Reply:
Hi,
thanks for all your suggestions.
How do you calculate 0xAF12 from the command output `echo -n "Plaid-Pad" | shasum` `af12462ebc2c6dd47492b3ad0e4be49f85e0949a`.

Reply:
Its just the first 4 characters, `echo -n "Plaid-Pad" | shasum | cut -c-4`

Reply:
Now i see it. :sweat_smile: